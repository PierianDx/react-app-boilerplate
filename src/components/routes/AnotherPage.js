import React from 'react';

const AnotherPage = () => <h1>Another Page</h1>;

export default AnotherPage;
