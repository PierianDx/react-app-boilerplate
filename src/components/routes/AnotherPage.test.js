import React from 'react';
import { shallow } from 'enzyme';

import AnotherPage from './AnotherPage';

describe('AnotherPage', () => {
  it('should render correctly', () => {
    const comp = shallow(<AnotherPage />);
    expect(comp).toMatchSnapshot();
  });
});
