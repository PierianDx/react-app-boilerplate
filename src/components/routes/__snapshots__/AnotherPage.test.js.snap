// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnotherPage should render correctly 1`] = `
<section
  className="section"
>
  <h1>
    Another Page
  </h1>
</section>
`;
